src/som/interpreter/actors/EventualSendNode.java

```diff
@@ -325,6 +339,14 @@ public final Object toNearRefWithoutResultPromise(final Object[] args) {
             current.getId());
       }
 
+      if (VmSettings.REPLAY) {
+        // TODO similar thing for any other usages of actor.send(). especially in timer prim
```

Comment:
What's this comment about? perhaps needs to be rephrased? What is "similar thing" referring to?

Reply:
before send is used, the send event needs to be consumed and the version number attached to the message. So if we have e.g. an http server that sends messages as callbacks, it has to do that. Exceptions to this rule are when a message already has the version number stored in it, e.g. due to some promise scenarios.

Reply:
hmm, I get your explanation, but not how it relates to the comment. What exactly needs to be implemented still?

Reply:
Essentially it's a note on what to do when implementing external event stuff. maybe I should remove the comment and make a note somewhere else

Reply:
Perhaps it can be rephrased to something like:
`// Note: When implementing external events, such as ... ...`